compose-builder/Makefile

```diff
@@ -1034,8 +1052,9 @@ pull: gen
 
 gen:
 	echo MQTT_VERBOSE=${MQTT_VERBOSE}
-	${DOCKER_COMPOSE}  -p edgex $(COMPOSE_FILES) ${GEN_COMMAND} > docker-compose.yml
-	rm -rf ./$(GEN_EXT_DIR)
+	echo "# Generated with: $(docker_ver)" > docker-compose.yml
+	${DOCKER_COMPOSE}  -p edgex $(COMPOSE_FILES) ${GEN_COMMAND} >> docker-compose.yml
+	if [ "$(NO_CLEANUP)" = "0" ]; then rm -rf ./$(GEN_EXT_DIR); fi
```

Comment:
Actually this is out of scope for this PR and will cause updates to all the generated files.

I like it, but it should be in a separate PR.

Reply:
ok. i reverted the docker version echo